docs/elements/guides/sign-in.mdx

```diff
@@ -0,0 +1,267 @@
+---
+description: Learn how to build a complete sign-in form with Clerk Elements.
+---
+
+# Build a sign-in flow
+
```

Comment:
```suggestion

<TutorialHero
  beforeYouStart={[
    {
      title: "Set up a Clerk application",
      link: "https://clerk.com/docs/quickstarts/setup-clerk",
    },
    {
      title: "Create a Next.js application",
      link: "https://nextjs.org/docs/getting-started/installation",
    },
    {
      title: "Upgrade to Core 2 if necessary",
      link: "https://clerk.com/docs/upgrade-guides/core-2/nextjs#upgrading-clerk-nextjs-to-core-2",
    },
  ]}
>

- Build a complete sign-in flow using Clerk Elements

</TutorialHero>

```

By adding this, we can omit the opening sentence and remove some text in the `<Callout>` below while making this guide look consistent with some of our other guides that have prerequisites 👍. Here's what it looks like:

![image](https://github.com/clerk/clerk-docs/assets/5156915/184c7652-f164-46a5-9b4e-c4abfc303e2a)